.circleci/config.yml

```diff
@@ -92,12 +92,6 @@ jobs:
       - run:
           name: "Run image snapshots"
           command: yarn test --image
-      - run:
-          name: Integration Test - Kichen sinks
-          command: yarn test --integration
-      - store_artifacts:
-          path: integration/__image_snapshots__
-          destination: integration_image_snapshots
```

Comment:
You can try to remove "Workaround for https://github.com/GoogleChrome/puppeteer/issues/290" step above as well

Reply:
And the corresponding script file: https://github.com/storybooks/storybook/blob/da23479536ad8e42d23dac0178cf8ff22f8be2d5/scripts/workaround-puppeteer-issue-290.sh

Reply:
uggh, sorry, done.